### ISSUE TITLE
NVDA not announcing aria-describedby on Chrome

### ISSUE THREAD

**Issue by svenka22** [nvaccess/nvda#9192]
[aria-describedbynot announced.zip](https://github.com/nvaccess/nvda/files/2788884/aria-describedbynot.announced.zip)

### Steps to reproduce:

- Go to https://login.microsoftonline.com on Chrome (this does not happen on Edge or other browsers)
- Have NVDA running
- Navigate to "Can't access your account link" using Tab key and activate it
- Also find attached 'aria-describedbynot announced.zip' which has a video for this issue.

### Actual behavior:

- Notice NVDA doesn't announce the heading/title of the page. It also doesn't announce the aria-label on the text

### Expected behavior:

- NVDA should announce the heading/title of the page. It should also announce the aria-label on the text

#### NVDA installed/portable/running from source:

- NVDA installed

#### NVDA version:

- 2018.4.1

#### Windows version:

- Windows 10 Enterprise

#### Does the issue still occur after restarting your PC?

- Yes

#### Have you tried any other versions of NVDA?

- No

**Comment by DrSooom** [comment]
@svenka22: Which Google Chrome version is installed?

**Comment by svenka22** [comment]
Version 72.0.3626.121 (Official Build) (64-bit)

**Comment by alanfluff** [comment]
+1 same [here (9534)](https://github.com/nvaccess/nvda/issues/9534).

`aria-labelledby` seems to work, though for IE it seems the ID of the container with the description needs `tabindex="-1"`; but it works, where `aria-describedby` seems to stubbornly fail.

**Comment by Adriani90** [comment]
I can reproduce this in any browser but only when pressing enter on the link from browse mode.
In Focus mode it works as expected.
cc: @MarcoZehe do you have any idea why NVDA does not report anything in Firefox?

**Comment by brennanyoung** [comment]
Using this snippet:
```
<div id="desc" hidden>
	<h2>This link shows</h2>
	<ul>
		<li>Maxium price</li>
		<li>Time to market</li>
		<li>Marketing budget</li>
	</ul>
</div>

<a href="#" aria-describedby="desc">Find out more info about this product line<a/>

```
When the link is in focus, NVDA+Chrome (both latest versions at time of posting) report "No Long Description" for `aria-describedby` if you use NVDA+D, but if you use "Report Current Object" (NVDA+numkeypad 5), it works ok. 

On FF, the description is announced when the link gets focus.

IMO, aria-describedby is intended for a long description, and should be announced with NVDA+D. Perhaps Chrome is building a screwy accessibility tree?

**Comment by Adriani90** [comment]
@brennanyoung using longdesc atribute would give you the ability to display the long description in a separate tab or window, via nvda+d, but this is not supported in every browser. See for example:
https://bugzilla.mozilla.org/show_bug.cgi?id=854848
From an user perspective, I would prefer to see such an implementation only for elements with a really long description (i.e. a diagram etc.).

Short description applied with aria describedby is already reported in focus mode.

However, it would be nice to have a reporting like for example "link downloads (has description)" when navigating with arrow keys in browse mode. If an user hears the note (has description) then it will be clear that he or she can press nvda+5 or nvda+shift+o in NVDA to get the description or to switch to focus mode to get it reported. @feerrenrut, @JulienCochuyt is it possible to implement something like this in browse mode?

**Comment by Adriani90** [comment]
Closing as duplicate of #9153 which contains much more details on aria specs.
